### PR TITLE
Slightly reduce length of copy in 'Reconnect to Your WiFi' dialog

### PR DESCRIPTION
**Pull request by angusjfw**
| Status  | Ticket/Issue |
| :---: | :--: |
| Ready/Hold | [Ticket](https://pi-top.atlassian.net/browse/<ticket-number>) |

#### Main changes
-

#### Screenshots (feature, test output, profiling, dev tools etc)

[insert screenshots here]

#### Other notes (e.g. implementation quirks, edge cases, questions / issues)
-

#### Manual testing tips
-

#### Tag anyone who definitely needs to review or help
-

### CODE REVIEW

File: frontend/src/pages/restartPage/connectivityWarningDialog/ConnectivityWarningDialog.tsx [hunk 1]
```diff
@@ -25,10 +25,10 @@ export const getContent = (shouldMoveAwayFromAp: boolean) => {
   if (shouldMoveAwayFromAp) {
     return (
       <>
-        Your pi-top is now connected to the Wi-Fi network you configured in a previous step, but your computer is still on the 'pi-top-XXXX' network.
+        Your pi-top was connected to a Wi-Fi network in a previous step, but your computer is still on the pi-top's hotspot.
```
Comment (olivierwilkinson) [inline frontend/src/pages/restartPage/connectivityWarningDialog/ConnectivityWarningDialog.tsx:28]: I think this could be shortened further:
'You connected your pi-top to Wi-Fi earlier, but your computer is still on the pi-top's WiFi hotspot.

**Review by olivierwilkinson — APPROVED** [review]
I added a suggestion that I think makes it a bit less wordy but it's definitely an improvement so happy to go ahead either way :)
